src/Nethermind/Nethermind.Network/P2P/ProtocolHandlers/ZeroNettyP2PHandler.cs

```diff
@@ -119,10 +119,12 @@ public override void ExceptionCaught(IChannelHandlerContext context, Exception e
             }
             else if (_session?.Node?.IsStatic != true)
             {
+                _session.InitiateDisconnect(DisconnectReason.Other, $"Exception in connection: {exception.GetType().Name} with message: {exception.Message}");
```

Comment:
Can you add a different `DisconnectReason` and use that instead? Like `DisconnectReason.NettyException`.

Reply:
Added Exception as reason